packages/vaadin-lumo-styles/mixins/required-field.js

```diff
@@ -10,17 +10,26 @@ import '../typography.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const requiredField = css`
+  :host {
```

Comment:
I suppose this is another place where we don't really need these private intermediate properties that we decided to remove from the helper and error stylesheets?

Reply:
Removed.